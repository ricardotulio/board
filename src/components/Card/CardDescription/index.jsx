export { default as CardDescription } from './CardDescription.jsx'
