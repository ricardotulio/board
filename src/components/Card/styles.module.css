.card {
  board: 1px solid black;
}
