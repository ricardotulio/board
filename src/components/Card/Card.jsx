import React from 'react'
import CardTitle from './CardTitle'
import CardDescription from './CardTitle'
import styles from './styles.module.css'

const Card = ({ card }) => (
  <div className={styles.card}>
    <CardTitle title={ card.title } />
    <CardDescription title={ card.description } />
  </div>
)

export default Card
