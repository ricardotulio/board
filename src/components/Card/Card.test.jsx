import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import Card from './Card'

it('can render a card', () => {
  const card = {
    title: 'card title',
    description: 'card description'
  }

  const { container, getByText } = render(<Card card={ card } />)
  expect(getByText('card title')).toBeInTheDocument()
  expect(getByText('card description')).toBeInTheDocument()
  expect(container.firstChild).toMatchSnapshot()
})
