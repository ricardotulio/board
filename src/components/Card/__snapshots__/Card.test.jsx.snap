// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render a card 1`] = `
<div
  class="card"
>
  <div
    class="card-title"
  >
    card title
  </div>
  <div
    class="card-title"
  >
    card description
  </div>
</div>
`;
